### PR TITLE
[12.0] add structure on loan + default structure

### PR DESCRIPTION
**Pull request by robinkeunen**
@remytms @houssine78 Need both of you for the reviews.

As usual, commit by commit makes it easier.

### CODE REVIEW

**Review by remytms — APPROVED** [review]
Nice

**Comment by remytms** [comment]
@houssine78 If you can review today, I can work on integrating structure in portal tomorrow morning. :)

File: investor_wallet_platform_base/models/coop.py [hunk 1]
```diff
@@ -5,9 +5,13 @@
...
 
-    structure = fields.Many2one('res.partner',
+    def default_structure(self):
+        return self.env.user.structure
+
+    structure = fields.Many2one(comodel_name='res.partner',
                                 string="Platform Structure",
-                                domain=[('is_plateform_structure', '=', True)])
+                                domain=[('is_plateform_structure', '=', True)],
+                                default=default_structure)
```
Comment (houssine78) [inline investor_wallet_platform_base/models/coop.py:14]: means that admin can't validate instead of amanager.

**Review by houssine78 — APPROVED** [review]

**Comment by robinkeunen** [comment]
@houssine78 done, désolé d'avoir bloqué.

**Comment by houssine78** [comment]
@robinkeunen no probs ;)

**Comment by remytms** [comment]
@robinkeunen Can it be merged ?

**Comment by remytms** [comment]
@robinkeunen Sorry, my tab was not up to date. ^^

**Comment by robinkeunen** [comment]
thought so :p

**Pull request merged**